frontend/src/service/utilities/Clipboard.service.ts

```diff
@@ -0,0 +1,46 @@
+export class ClipboardService {
+  private fallbackCopyTextToClipboard(text: string): boolean {
+    let textArea = document.createElement('textarea');
+
+    // set minimal visual effects in case textarea flashes
+    textArea.style.position = 'fixed';
+    textArea.style.top = '0';
+    textArea.style.left = '0';
+
+    textArea.style.width = '2em';
+    textArea.style.height = '2em';
+
+    textArea.style.padding = '0';
+    textArea.style.border = 'none';
+    textArea.style.outline = 'none';
+    textArea.style.boxShadow = 'none';
+    textArea.style.background = 'transparent';
```

Comment:
Can this be wrapped within a private function called `createInvisibleTextArea()`?

Reply:
Okay, this looks good.